website/content/docs/agent/caching/persistent-caches/index.mdx

```diff
@@ -1,9 +1,71 @@
 ---
 layout: docs
-page_title: Vault Agent Persistent Cache Types
-description: Persistent Cache Types for Vault Agent Caching
+page_title: Vault Agent Persistent Caching
+description: Vault Agent Caching
 ---
 
-# Vault Agent Persistent Cache Types
+# Vault Agent Persistent Caching
+
+Vault Agent can restore tokens and leases from a persistent cache file created
+by a previous Vault Agent process. The persistent cache is a BoltDB file that
+includes tuples encrypted by a generated encryption key. The encrypted tuples
+include the Vault token used to retrieve secrets, leases for tokens/secrets, and
+secret values.
+
+-> **Note:** Vault Agent Caching persistent cache will only restore _leased_
+secrets. Secrets that are not renewable, such as KV v2, will not be persisted.
+
+In order to use Vault Agent persistent cache, auto-auth must be used. During the
+restoration of the cache, Vault Agent will pre-populate auto-auth with the
+persisted token. This token is required to renew restored leases. If the token
+has expired, the cached leases will be evicted and secrets will need to be
+retrieved from Vault.
+
+If Vault Agent templating is enabled alongside of the persistent cache, Vault
+Agent will automatically route templating requests through the cache. This
+ensures template requests are cached and restored properly.
+
+At the time of this writing, Vault Agent persistent cache is only supported in a
+Kubernetes environment. In the future the persistent cache will be expanded to
+include other environments.
+
+
+## Vault Agent Persistent Cache Types
 
 Please see the sidebar for available types and their usage/configuration.
+
+## Persistent Cache Example Configuration
+
+An example configuration, with very contrived values, follows:
```

Comment:
Suggestion:
Here is an example of a persistent cache configuration.

Reply:
Addressed in [eb64fb9](https://github.com/hashicorp/vault/pull/15195/commits/eb64fb945c55953276443c88fcc87f1a662f00a9)